Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -2439,6 +2591,136 @@ def get_papi_edgehostname_creation_status_command_ec(raw_response: dict) -> Tupl
     return entry_context, human_readable
 
 
+# Created by D.S. 2022-11-25
+def get_papi_property_bygroup_ec(raw_response: dict) -> Tuple[list, list]:
+    """
+        Parse papi property
+    Args:
+        raw_response:
+    Returns:
+        dictionary of Property
+    """
+    entry_context = []
+    human_readable = []
+    if raw_response:
+        entry_context.append(assign_params(**{
+            "AccountId": raw_response["accountId"],
+            "ContractId": raw_response["contractId"],
+            "GroupId": raw_response["groupId"],
+            "PropertyId": raw_response["propertyId"],
+            "PropertyName": raw_response["propertyName"],
+            "LatestVersion": raw_response["latestVersion"],
+            "StagingVersion": raw_response["stagingVersion"],
+            "ProductionVersion": raw_response["productionVersion"],
+            "AssetId": raw_response["assetId"]
+        }))
+        human_readable = entry_context
+    return entry_context, human_readable
+
+
+# Created by D.S. 2023-02-27
```

Comment:
Please remove comment

Reply:
We have two individuals working on the same integration and only using the Xsoar's Web based development interface.
We are planning to get the CI/CD pipeline setup within this year. However, before that is ready, we are probably going to keep track of some of our work in this manner.